packages/spatial/src/renderer/components/PostProcessingComponent.tsx

```diff
@@ -23,29 +23,75 @@ All portions of the code written by the Ethereal Engine team are Copyright © 20
 Ethereal Engine. All Rights Reserved.
 */
 
-import { useEffect } from 'react'
-
-import { defineComponent, useComponent, useEntityContext } from '@etherealengine/ecs'
-import { NO_PROXY_STEALTH } from '@etherealengine/hyperflux'
-
-import { defaultPostProcessingSchema, EffectPropsSchema } from '../effects/PostProcessing'
-import { configureEffectComposer } from '../functions/configureEffectComposer'
+import { Entity, defineComponent, useComponent, useEntityContext } from '@etherealengine/ecs'
+import { ErrorBoundary, NO_PROXY, getState, useHookstate } from '@etherealengine/hyperflux'
+import { Effect, EffectComposer, EffectPass, OutlineEffect, RenderPass } from 'postprocessing'
+import React, { Suspense, useEffect } from 'react'
+import { MotionBlurEffect, SSGIEffect, SSREffect, TRAAEffect } from 'realism-effects'
+import { ArrayCamera, Scene, WebGLRenderer } from 'three'
+import { CameraComponent } from '../../camera/components/CameraComponent'
+import { HighlightState } from '../HighlightState'
+import { RendererState } from '../RendererState'
+import { RendererComponent } from '../WebGLRendererSystem'
+import { ObjectLayers } from '../constants/ObjectLayers'
+import { PostProcessingEffectState } from '../effects/EffectRegistry'
+import { LinearTosRGBEffect } from '../effects/LinearTosRGBEffect'
 import { useScene } from './SceneComponents'
 
+declare module 'postprocessing' {
+  interface EffectComposer {
+    // passes
+    EffectPass: EffectPass
+    // effects
+
+    BloomEffect: BloomEffect
+    BrightnessContrastEffect: BrightnessContrastEffect
+    ChromaticAberrationEffect: ChromaticAberrationEffect
+    ColorAverageEffect: ColorAverageEffect
+    ColorDepthEffect: ColorDepthEffect
+    DepthOfFieldEffect: DepthOfFieldEffect
+    DotScreenEffect: DotScreenEffect
+    FXAAEffect: FXAAEffect
+    GlitchEffect: GlitchEffect
+    //GodRaysEffect: GodRaysEffect
+    GridEffect: GridEffect
+    HueSaturationEffect: HueSaturationEffect
+    LensDistortionEffect: LensDistortionEffect
+    LinearTosRGBEffect: LinearTosRGBEffect
+    LUT1DEffect: LUT1DEffect
+    LUT3DEffect: LUT3DEffect
+    MotionBlurEffect: MotionBlurEffect
+    NoiseEffect: NoiseEffect
+    PixelationEffect: PixelationEffect
+    ScanlineEffect: ScanlineEffect
+    ShockWaveEffect: ShockWaveEffect
+    SSAOEffect: SSAOEffect
+    SSREffect: SSREffect
+    SSGIEffect: SSGIEffect
+    TextureEffect: TextureEffect
+    TiltShiftEffect: TiltShiftEffect
+    ToneMappingEffect: ToneMappingEffect
+    TRAAEffect: TRAAEffect
+    VignetteEffect: VignetteEffect
+
+    SMAAEffect: SMAAEffect
+    OutlineEffect: OutlineEffect
+  }
+}
+
 export const PostProcessingComponent = defineComponent({
   name: 'PostProcessingComponent',
   jsonID: 'EE_postprocessing',
 
   onInit(entity) {
     return {
       enabled: false,
-      effects: defaultPostProcessingSchema
+      effects: {} as Record<string, any> // effect name, parameters
```

Comment:
can we add a better type than `any` for the record value here? we could a base type without effect-specific props perhaps for at least the context of this module

Reply:
implemented